error-issue/test/issue_builder.test.ts

```diff
@@ -83,6 +85,99 @@ describe('IssueBuilder', () => {
     });
   });
 
+  describe('possibleAssignees', () => {
+    const blames = [
```

Comment:
Consider making this a .json fixture file

Reply:
I considered it, but I leaned toward including it here so that someone looking at the test can quickly look at the underlying values and see ie. author, dates, changed files, etc. Instead I added a helper that takes only the relevant fields so it's a bit easier to look at/parse.